nimble/transport/syscfg.yml

```diff
@@ -29,6 +29,7 @@ syscfg.defs:
             - native
             - dialog_cmac
             - nrf5340
+            - ipc
```

Comment:
`icbmsg` or `ipc_icbmsg` - I think the latter is more preferable as we may later opt rename nrf5340 and cmac to include `ipc_` prefix as well -> @sjanc ?